server.go

```diff
@@ -252,6 +276,27 @@ func (s *Server) HandleMetricPacket(packet []byte) {
 	}
 }
 
+func (s *Server) HandleTracePacket(packet []byte) {
+	// Unlike metrics, protobuf shouldn't have an issue with 0-length packets
+	if len(packet) == 0 {
+		log.Error("received zero-length trace packet")
+	}
+
+	// Technically this could be anything, but we're only consuming trace spans
+	// for now.
+	newSample := &ssf.SSFSample{}
+	err := proto.Unmarshal(packet, newSample)
+	if err != nil {
+		log.Fatal("Trace unmarshaling error: ", err)
+	}
+
+	s.statsd.Count("veneur.traces.handled", 1, []string{}, 1.0)
+
+	log.WithField("proto", proto.CompactTextString(newSample)).Debug("Handling trace packet")
```

Comment:
i don't remember how expensive logrus's debug logs are, but we probably shouldn't be logging from a hot loop for unexceptional situations